getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,238 @@
+---
+layout: default
+title: Installing Community
+published: true
+sorting: 80
+---
+
+These instructions describe how to download and install the latest version of CFEngine Community in a docker containerized environment using pre-compiled rpm packages for ubi9 images.
+
+It also provides instructions for the following:
+
+* **Install CFEngine on a policy server (hub) and on a Host (client).**
+A Policy Server (hub) is a CFEngine instance that contains promises (business policy) that get deployed to Hosts.
+Hosts are clients that retrieve and execute promises.
+
+* **Bootstrap the policy server to itself and then bootstrap the Host(s) to the Policy Server.**
+Bootstrapping establishes a trust relationship between the Policy Server and all Hosts. Thus, business policy that you create in the Policy Server can be deployed to Hosts throughout your company.
+
+Bootstrapping completes the installation process.
+
+<hr>
+
+This guide describes how to set up a client-server model with CFEngine and, through policy, manage both containers.
+Docker contaiers will be created, one container to be the Policy Server (server), and another container that will be the Host Agent (client).
+
+Both will will run ubi9 images and communicate on a container network.
+Upon completion, you are ready to start working with CFEngine.
+
+## Requirements
+* 1G+ disk space
+* 1G+ memory
+* Working [Docker Engine](https://docs.docker.com/engine/) or [Podman](https://podman.io/) setups on a supported [x86_64](https://en.wikipedia.org/wiki/X86-64) platform.
+
+**Note**: This document considers [Docker Engine](https://docs.docker.com/engine/) for all examples.</br>
+Use of [Podman](https://podman.io/) shall be similar with adequate adaptations. (_Ref_: [Emulating Docker CLI with Podman](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman)).
+
+## Overview
+1. Installing container engine
+2. Preparing CFEngine hub in container
+3. Preparing CFEngine host in container
+4. Using docker compose</br>
+    4.1. Preparing container image for CFEngine</br>
+    4.2. Using docker compose service</br>
+5. Supporting notes and references</br>
+    5.1. Enabling systemd for WSL</br>
+    5.2. References</br>
+
+## Installing container engine
+_Ref_: [Install Docker Engine](https://docs.docker.com/engine/install/)</br>
+OR</br>
+_Ref_: [Podman Installation Instructions](https://podman.io/docs/installation)
+(_Optionally_: [Emulating Docker CLI with Podman](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman))
+
+## Preparing CFEngine hub in container
+1. Run the container with systemd
+    ```
+    $ docker run --privileged -dit --name=cfengine-hub registry.access.redhat.com/ubi9-init /usr/sbin/init
+    ```
```

Comment:
These do not need to be lists, and should use our `command` blocks. See below for examples:

https://docs.cfengine.com/docs/3.24/getting-started-writing-policy.html

https://github.com/cfengine/documentation/edit/3.24/getting-started/writing-policy.markdown